Assignment2/mcaminome-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,62 @@
+{
+	"@context": {
+  	"people": "http://www.oeg-upm.net/ontologies/people",
+    "fullname": "people:fullname",
+    "hometown": "people:hometown",
+    "age": "people:age",
+    "almamater":"people:almamater",
+    "parents":"people:parents",
+    "mother":"parents:mother",
+    "father":"parents:father",

```

Comment:
These attribute definitions work, but semantically they aren't ok. The best way is to use a prefix and add the name of the concept/property you want to use e.g. "mother":"yourvocab:mother".
The same comment applies to the combination you did in the subject attributes definition. 
You don't need to change them, but remember to provide good definitions for the future.